docs/UserGuide.md

```diff
@@ -110,6 +111,36 @@ The `-h` or `--hashes` argument configures BinSkim to emit MD5, SHA1 and SHA256
 
 The `-e` or `--environment` argument configures BinSkim to emit machine environment details to the SARIF log file specified via the -o argument. This information includes the user account, machine name, working directory and complete set of environment variables and definitions that were present during the analysis run. This information may be useful in some tool troubleshooting scenarios. WARNING: the information emitted by the -e argument may represent unwanted information disclosure.
 
+#### --rich-return-code
```

Comment:
>rich [](start = 7, length = 4)

Good stuff.  We could also document the Fatal and NonFatal enum values. This will be useful when contriving result code checks in things like batch files/powershell scripts.